openpgp/s2k/s2k.go

```diff
@@ -169,25 +246,48 @@ func Iterated(out []byte, h hash.Hash, in []byte, salt []byte, count int) {
 	}
 }
 
+func memoryExpToKibibytes(memoryExp uint8) uint32 {
+	return (uint32(1) << memoryExp)
+}
+
+// Argon2Derive writes to out the key derived from the password (in) with the Argon2
+// function (RFC 4880, section 3.7.1.4)
+func Argon2Derive(out []byte, in []byte, salt []byte, passes uint8, paralellism uint8, memoryExp uint8) {
```

Comment:
Since none of the other functions have "derive" in the name I would just call this "Argon2". Also, since it's from the crypto refresh rather than RFC 4880;
```suggestion
// Argon2 writes to out the key derived from the password (in) with the Argon2
// function (crypto refresh, section 3.7.1.4)
func Argon2(out []byte, in []byte, salt []byte, passes uint8, paralellism uint8, memoryExp uint8) {
```